airflow_home/dags/dag.py

```diff
@@ -0,0 +1,80 @@
+import os
+import sys
+from datetime import datetime, timedelta
+
+import yaml
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+
+#  Need to make scripts package visible
+sys.path.append(os.environ.get('AIRFLOW_HOME'))
+
+from scripts.extract import extract
+from scripts.load import data_load
+from scripts.transform import find_result_data
+
+
+def load_config_from_yaml(file_path: str) -> dict():
+    with open(file_path, 'r') as f:
+        config = yaml.load(f, Loader=yaml.FullLoader)
+
+    return config
+
+
+default_args = {
+    'depends_on_past': False
+}
+
+
+def create_etl_tasks(
+    dag: DAG,
+    ticker: str,
+    task_before: PostgresOperator
+) -> None:
+    task_1 = PythonOperator(
+        task_id=f'extract_{ticker}',
+        python_callable=extract,
+        provide_context=True,
+        op_args=(ticker, ),
+        )
+
+    task_2 = PythonOperator(
+        task_id=f'transform_{ticker}',
+        python_callable=find_result_data,
+        provide_context=True,
+        op_args=(ticker, ),
+        )
+
+    task_3 = PythonOperator(
+        task_id=f'load_{ticker}',
+        python_callable=data_load,
+        provide_context=True,
+        op_args=(ticker, ),
+        )
+
+    task_before.set_downstream(task_1)
+    task_1.set_downstream(task_2)
+    task_2.set_downstream(task_3)
+
+
+with DAG(
+    'tickers_to_postgres',
+    default_args=default_args,
+    description='Tickers from API loads into PostgreSQL',
+    schedule_interval=timedelta(hours=2),
+    start_date=datetime(2021, 12, 2, 0, 0, 0),
+    catchup=False
+) as dag:
+    task = PostgresOperator(
+        task_id='create_table',
+        postgres_conn_id='postgres_default',
+        sql="""create table if not exists ticker_info (TICKER character \
+               varying (4) not null, ASK money not null, BID money not null, \
+               DATETIME_GATHERED timestamp not null) ;""",
```

Comment:
Speaking of the SQL casing there.

There is a good rule of thumb: if you are writing some SQL code in IDE with syntax highlighting specifically for it (e.g. you are creating SQL queries in .sql files), you can use lowercase everywhere.

But if you are using SQL inside python (or other code), where the syntax highlighting doesn't work for SQL specifically, you should put general SQL statements to upper case (e.g. `SELECT`, `FROM` and other keywords), and other strings such as column names, tables names, query parameters should be lowercased.

Reply:
Also, here you can make table name and column names configurable by formatting the SQL string before creating the operator itself.

E.g:

1. Create two global constants named `TABLE_NAME`, `COLUMNS_INFO`. The first one will be a string, the second one will be a dictionary with format `{ col_name : col_type }`.
2. Use Python to create a string based on these two constants: `ticker_info (ticker CHARACTER VARYING (4), ask MONEY NOT NULL, bid MONEY NOT NULL, datetime_gathered TIMESTAMP NOT NULL)`.
3. Create the query: `f"CREATE TABLE IF NOT EXISTS {TABLE_NAME} {column_info_string}"` and use it in Postgres operator.

This will give you better configurability of your job.

Reply:
And one more thing:

It is generally discouraged to use the triple-quoted strings anywhere apart from documentation strings.

Alternatives are:

```
Pay attention to no commas here:

long_string = (
    "Hello, world "
    "Some more things "
    "Other "
)

long_string = "Hello, world! " \
              "Something!"
```